STACpopulator/populator_base.py

```diff
@@ -144,9 +143,12 @@ def publish_stac_collection(self, collection_data: dict[str, Any]) -> None:
         post_stac_collection(self.stac_host, collection_data, self.update, session=self._session)
 
     def ingest(self) -> None:
+        counter = 0
         LOGGER.info("Data ingestion")
-        for item_name, item_data in self._ingest_pipeline:
-            LOGGER.info(f"Creating STAC representation for {item_name}")
+        for item_name, item_loc, item_data in self._ingest_pipeline:
+            LOGGER.info(f"New data item: {item_name}")
+            if item_loc:
+                LOGGER.info(f"Data location: {item_loc}")
```

Comment:
The `if` can be removed once the paths are returned for directory loader.

Reply:
Done.